pkg/capture/capture.go

```diff
@@ -219,7 +219,7 @@ func (c *Capture) process() <-chan error {
 		defer c.wgProc.Done()
 
 		// Main packet capture loop which an interface should be in most of the time
-		localBuf := NewLocalBuffer(c.captureHandle, WithSizeLimit(c.config.LocalBufferSizeLimit))
```

Comment:
Size limit is now governed globally (via `setLocalBuffers()`, respectively by reasonable defaults)